x-pack/plugins/ingest_manager/server/routes/agent/index.ts

```diff
@@ -42,6 +45,35 @@ import { postNewAgentActionHandlerBuilder } from './actions_handlers';
 import { appContextService } from '../../services';
 import { postAgentsUnenrollHandler } from './unenroll_handler';
 
+const ajv = new Ajv({
+  coerceTypes: true,
+  useDefaults: true,
+  removeAdditional: true,
+  allErrors: false,
+  nullable: true,
+});
+
+function schemaErrorsText(errors: any, dataVar: any) {
```

Comment:
Can we replace the `any`? e.g. 
```suggestion
function schemaErrorsText(errors: Ajv.ErrorObject[], dataVar: any) {
```